### PR TITLE
Clean up devops handbook content

### PR DESCRIPTION
**Pull request by michaellzc**
Some clean up and borrow some ideas from the delivery handbook

### CODE REVIEW

File: content/departments/product-engineering/engineering/cloud/devops/index.md [hunk 1]
```diff
@@ -27,28 +27,58 @@ This team is responsible for
...
+
+## How we work
+
+### Issue tracking
+
+[The DevOps GitHub Project board](https://github.com/orgs/sourcegraph/projects/220/views/23) is the single source of truth.
+
+### Planning, Sync & Retro
 
-## How to contact the team and ask for help
```
Comment (daxmc99) [inline content/departments/product-engineering/engineering/cloud/devops/index.md:45]: Maybe we should mention the slack support handle here?

**Review by daxmc99 — COMMENTED** [review]
LGTM

**Review by daxmc99 — APPROVED** [review]

File: content/departments/product-engineering/engineering/cloud/devops/index.md (hunk 1 above)
Comment (michaellzc) [reply content/departments/product-engineering/engineering/cloud/devops/index.md:45]: I moved it to the top for better visibility to other team https://github.com/sourcegraph/handbook/pull/3335/files#diff-1f6d0f991bbf3a06d1d5a82f14e8e408601bb1e5a9ce3f2ebd9ae6d84a571f94R30-R34

wdyt?

File: content/departments/product-engineering/engineering/cloud/devops/index.md [hunk 2]
```diff
@@ -27,28 +27,58 @@ This team is responsible for
 
 {{generator:product_team.devops}}
 
+## Contact & Support Guideline
+
+- The best way to contact the `cloud-devops` team is in the #cloud-devops slack channel. If it is urgent, please cc `@devops-support` in your message.
+- Issues with `team/devops` labels and [@team/cloud-devops](https://github.com/orgs/sourcegraph/teams/cloud-devops) team on GitHub
+- Please do not `@cloud-devops-team` or directly message individual teammates - this is to try and protect their focus
```
Comment (michaellzc) [inline content/departments/product-engineering/engineering/cloud/devops/index.md]:
```suggestion
## Contact & Support Guideline

- The best way to contact the `cloud-devops` team is in the #cloud-devops slack channel. If it is urgent, please cc `@devops-support` in your message.
- Issues with `team/devops` labels and [@team/cloud-devops](https://github.com/orgs/sourcegraph/teams/cloud-devops) team on GitHub
- Please do not `@cloud-devops-team` or directly message individual teammates - this is to try and protect their focus. Instead, use the `@devops-support` handle, and the On-Call DevOps teammate will be notified.
```

File: content/departments/product-engineering/engineering/cloud/devops/index.md (hunk 1 above)
Comment (daxmc99) [reply content/departments/product-engineering/engineering/cloud/devops/index.md:45]: Looks good!

**Pull request merged**